crates/model/src/instruments/mod.rs

```diff
@@ -190,109 +374,108 @@ pub const EXPIRING_INSTRUMENT_TYPES: [InstrumentClass; 4] = [
 mod tests {
     use rstest::rstest;
 
-    use crate::instruments::{CurrencyPair, Instrument, stubs::*};
+    use super::*;
+    use crate::{instruments::stubs::*, types::Money};
 
     #[rstest]
-    fn test_make_qty_standard_rounding(currency_pair_btcusdt: CurrencyPair) {
+    fn make_qty_rounding(currency_pair_btcusdt: CurrencyPair) {
```

Comment:
We'll want to use `rstest` cases here for the parametrization. We could have substantially more cases as well.